examples/examples3d.jl

```diff
@@ -625,8 +625,61 @@
         streamplot(f, -1.5..1.5, -1.5..1.5, -1.5..1.5, colormap = :magma, gridsize = (10, 10), arrow_size = 0.06)
     end
 
+    @cell "Fractional Brownian surface" ["3d"] begin
```

Comment:
```suggestion
    @cell "Brownian surface" ["3d"] begin
```

Reply:
Dang...this will have to go in the next refimage update.  Will keep it in mind!